Jenkinsfile

```diff
@@ -46,6 +46,11 @@ git merge ${merge_opts} '${commit}' -m '${commit_msg}'
 """)
 }
 
+// https://issues.jenkins-ci.org/browse/JENKINS-37069
+def fix_workspace_label() {
+    sh "chcon -Rt svirt_sandbox_file_t '${env.WORKSPACE}'*"
```

Comment:
It is better to use the `:z` (note, lowercase) bindmount flag for this, if possible.  The name of `svirt_sandbox_file_t` is changing soon, so if we hard-code the SELinux type, we'll have to update it when the policy is updated.